tests/test_driver.py

```diff
@@ -5,7 +5,7 @@
 import unittest
 import requests
 import tempfile
-
+import mintapi.constants as constants
```

Comment:
Nice, looks much cleaner! Do you mind just updating this to the "from mintapi import constants" as well for consistency?